cylc/flow/network/schema.py

```diff
@@ -1639,7 +1648,7 @@ class Arguments:
 class Reload(Mutation):
     class Meta:
         description = sstrip('''
-            Reload the configuration of a running workflow.
+            Reload the configuration of a workflow.
```

Comment:
I'll put this one back so it appears in the short description.
```suggestion
            Reload the configuration of a running workflow.
```